plugins/example/example.coffee

```diff
@@ -0,0 +1,22 @@
+(() ->
+  class HelloWorldPlugin
```

Comment:
ok, i'm fine with that.  why require registering the class at all, then?   should we just have them pass a plain object?